.travis.yml

```diff
@@ -35,7 +35,7 @@ install:
   - echo "SCALA VERSION  $TRAVIS_SCALA_VERSION"
   - echo "LATEST VERSION $LATEST"
   - export CURRENT=${TRAVIS_SCALA_VERSION}-${KAFKA_VERSION}
-  - docker build --build-arg kafka_version=$KAFKA_VERSION --build-arg scala_version=$TRAVIS_SCALA_VERSION -t wurstmeister/kafka .
+  - docker build --build-arg kafka_version=$KAFKA_VERSION --build-arg scala_version=$TRAVIS_SCALA_VERSION --build-arg vcs_ref=$(git rev-parse --short HEAD) -t wurstmeister/kafka .
```

Comment:
I think this is probably available from travis-ci env (https://docs.travis-ci.com/user/environment-variables/#default-environment-variables)